docs/commandline/create.md

```diff
@@ -0,0 +1,56 @@
+# create
+
+create command is used to create a container according to user's configuration.
+
+## Description
+
+The `pouch create` command creates a container object in Pouchd. When creating, all configuration user input will be stored in memory store of Pouchd. This is useful when you hope to create a container configuration ahead of time so that Pouchd will preserve the resource in advance. The container you created is ready to start when you need it.
+
+## Help Information
+
+``` shell
+$ pouch create --help
+Create a new container with specified image
+
+Usage:
+  pouch create [image] [flags]
+
+Flags:
+  -h, --help             help for create
+      --name string      specified the container's name
+  -t, --tty              allocate a tty device
+  -v, --volume strings   create container with volumes
+
+Global Flags:
+  -H, --host string        Specify listen address of pouchd (default "unix:///var/run/pouchd.sock")
+      --tlscacert string   Specify CA file of tls
+      --tlscert string     Specify cert file of tls
+      --tlskey string      Specify key file of tls
+      --tlsverify          Switch if verify the remote when using tls
+```
+
+## Flag Guidance
+
+### --name
+
+User can set container's name by passing flag `--name`. For example:
+
+``` shell
+$ pouch create --name foo busybox:latest
```

Comment:
This has an issue which is described in https://github.com/alibaba/pouch/issues/175